lib/model/Tokens.js

```diff
@@ -95,6 +95,34 @@ Tokens.prototype.connectByApiToken = function(tenantName, apiToken, callback) {
     });
 }
 
+/**
+ * Authenticate with the API endpoint to get the entire token object
+ * 
+ * @param {string} tenantName - name of the tenant we want to access
+ * @param {string} apiToken - the apiToken retrieved from mPulse UI
+ * @param {function} callback - 
+ * [Called after the API Auth request was finalized]{@link Tokens~getToken(callback)}
+ */
+Tokens.prototype.getToken = function(tenantName, apiToken, callback) {
```

Comment:
@cratter49 This isn't an API token, so it shouldn't be called `apiToken`.  Just `token`.

Reply:
@msolnit Well in this case it's both an API - Authentication token being used to make a request to the server in addition to being used as the ID of the object to find.

Reply:
No it's not.  This method will only work after you've already authenticated via `connect()`, right?

Reply:
neither `tenantName` nor `apiToken` are required (or have any meaning) for this call, but `token` (ie, the thing returned by `connect*` is required.